frontend/packages/process-editor/src/components/ConfigPanel/EditDataType/SelectDataType/SelectDataType.tsx

```diff
@@ -42,12 +43,12 @@ export const SelectDataType = ({
         <option disabled={true} value={'noModelKey'}>
           {t('process_editor.configuration_panel_select_datamodel')}
         </option>
-        {dataModelIds.map((option) => (
+        {datamodelIds.map((option) => (
           <option key={option} value={option}>
             {option}
           </option>
         ))}
-      </NativeSelect>
+      </StudioNativeSelect>
       <div className={classes.buttons}>
         <StudioButton
```

Comment:
Maybe we should add some custom css here to place the close button next to the select if the `hideDeleteButton` is true? No biggie if skipped tho, but I think it would make the design a little bit nicer :D

Reply:
Done 😄 
<img width="290" alt="image" src="https://github.com/Altinn/altinn-studio/assets/133344438/6638e830-4b52-4529-9466-922471269582">